providers/flagsmith/example_test.go

```diff
@@ -0,0 +1,56 @@
+package flagsmith_test
+
+import (
+	"context"
+	"fmt"
+
+	flagsmithClient "github.com/Flagsmith/flagsmith-go-client/v4"
+	flagsmith "github.com/open-feature/go-sdk-contrib/providers/flagsmith/pkg"
+	"github.com/open-feature/go-sdk/openfeature"
+)
+
+
+func Example() {
+	provider := flagsmith.NewProvider(
+		// See https://docs.flagsmith.com/clients/server-side?language=go
+		flagsmithClient.NewClient(
+			// Local evaluation is the recommended mode for server-side applications.
+			// Client-side applications should use remote evaluation.
+			// See https://docs.flagsmith.com/clients/
+			"ser.your-server-side-sdk-key",
+			flagsmithClient.WithLocalEvaluation(ctx),
```

Comment:
```suggestion
			flagsmithClient.WithLocalEvaluation(context.TODO()),
```

Reply:
Fixed in https://github.com/open-feature/go-sdk-contrib/pull/715/commits/95e480932f30b76794b2f21de323320f95fc49b4, thanks!